java/clients/src/main/java/sleeper/clients/QueryWebSocketCommandLineClient.java

```diff
@@ -70,7 +70,11 @@ protected void init(TableProperties tableProperties) {
 
     @Override
     protected void submitQuery(TableProperties tableProperties, Query query) {
-        queryWebSocketClient.submitQuery(query);
+        try {
+            queryWebSocketClient.submitQuery(query).join();
+        } catch (InterruptedException e) {
+            out.println("Failed to run query: " + e.getMessage());
```

Comment:
An InterruptedException should cause the thread to terminate immediately. This could rethrow and re-interrupt the thread.

Reply:
Thanks, i'm rethrowing this exception out of the method